templates/admin/change_list.html

```diff
@@ -0,0 +1,93 @@
+{% extends "admin/base_site.html" %}
```

Comment:
@SivikGosh @lemontree210  I believe there should be a better solution otherwise I would put this task to the backlog since it requires too much effort. I think this violates the core idea of django which is to have everything out of the box and reuse pre-created/default components as much as possible since this is the only reason django exists (to save time and effort while ensuring quality) and if such a small task requires so many changes in these components I do not think it is worth it.

Reply:
> I think this violates the core idea of django which is to have everything out of the box and reuse pre-created/default components as much as possible since this is the only reason django exists.

I can only agree to a certain extent. Yes, Django strives to provide _basic/standard_ components out of the box, but I haven't heard of users being discouraged from e.g. editing templates because this would violate Django's philosophy. I think even the official docs contain info on how to customize admin site by changing its templates.